src/main/java/lotto/model/Lottos.java

```diff
@@ -0,0 +1,50 @@
+package lotto.model;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Lottos {
+    private List<Lotto> lottos;
+
+    private static final int DEFAULT_LOTTO_PRICE = 1000;
+    private static final int LOTTO_LIMIT_NUMBER = 45;
+    private static final int LOTTO_LIMIT_SIZE = 6;
+
+    public Lottos() {}
+    public Lottos(List<Lotto> lottos) {
+        this.lottos = lottos;
+    }
+    public Lottos(final int money) {
+        this.lottos = new ArrayList<>();
+        for (int i = 0; i < purchaseLottoCount(money); i++) {
+            lottos.add(new Lotto(createRandomList()));
+        }
+    }
+
+    static public List<Result> matches(Lotto winningLotto, int bonus, Lottos myLotto) {
+        return myLotto.getLottos().stream()
+                .map(lotto -> lotto.match(winningLotto, bonus))
+                .collect(Collectors.toList());
+    }
```

Comment:
static일 필요가 없을 것 같아요! 
아래 코드를 참고해보세요 :)

```suggestion
    public List<Result> matches(Lotto winningLotto, int bonus) {
        return lottos.stream()
                .map(lotto -> lotto.match(winningLotto, bonus))
                .collect(Collectors.toList());
    }
```